packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -159,9 +159,9 @@ export class ComponentShowcase extends ClassComponent {
         <div class="basic-gallery">
           <CWText type="h3">Tag</CWText>
```

Comment:
As discussed, this is a SmartUI pattern which we should be moving away from @zakhap @affordances